__tests__/app.test.js

```diff
@@ -70,3 +70,49 @@ describe("GET-/api/reviews", () => {
       });
   });
 });
+
+describe.only("/api/reviews/:review_id", () => {
+  test("GET - 200: Returns the review with the correct ID", () => {
+    return request(app)
+      .get("/api/reviews/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          review_id: 3,
+          title: "Ultimate Werewolf",
+          review_body: "We couldn't find the werewolf!",
+          designer: "Akihisa Okui",
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          votes: 5,
+          category: "social deduction",
+          owner: "bainesface",
+          created_at: "2021-01-18T10:01:41.251Z",
+        });
+      });
+  });
+  test("GET - 404: Route not found", () => {

```

Comment:
This test is not tied specifically to this endpoint, instead applying to every possible endpoint that doesn't exist. I'd move it to a separate describe block and have that serve as a test for all endpoint typos